utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/WebSecurityConfig.java

```diff
@@ -8,6 +8,8 @@
 public class WebSecurityConfig extends WebSecurityConfigurerAdapter{
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http.csrf().disable().authorizeRequests().anyRequest().permitAll();
+        http.csrf().disable().authorizeRequests().anyRequest().permitAll().and()
```

Comment:
This is pretty hard to read right now, might I suggest:

```java
        http
            .authorizeRequests().anyRequest().permitAll()
            .and()
            .csrf().disable()
            .headers().disable();
```

Reply:
Applied